src/main/java/codechicken/core/launch/DepLoader.java

```diff
@@ -508,7 +508,6 @@ private void scanDepInfo(File file) {
                 zip.close();
             } catch (Exception e) {
                 System.err.println("Failed to load dependencies.info from " + file.getName() + " as JSON");
-                e.printStackTrace();
```

Comment:
Do not suppress this error output. It makes it impossible to debug if something went wrong. IIRC dependencies.json is used by more than just code chicken mods